androidApp/src/main/java/com/EENX15_22_17/digital_journal/android/screens/triage/somatichealth/SomaticHealthScreen.kt

```diff
@@ -0,0 +1,193 @@
+package com.EENX15_22_17.digital_journal.android.screens.triage.somatichealth
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.grid.GridCells
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import com.EENX15_22_17.digital_journal.android.R
+import com.EENX15_22_17.digital_journal.android.dataModel.NursesNeeds
+import com.EENX15_22_17.digital_journal.android.dataModel.YesNo
+import com.EENX15_22_17.digital_journal.android.dataModel.nursesNeeds
+import com.EENX15_22_17.digital_journal.android.dataModel.yesNoLabels
+import com.EENX15_22_17.digital_journal.android.ui.DetailPageWrapper
+import com.EENX15_22_17.digital_journal.android.ui.components.*
+import com.EENX15_22_17.digital_journal.android.ui.theme.Colors
+
+@Composable
+fun HealthHistoryPage(
```

Comment:
The name of the Composable does not match the new file-name

Reply:
Fixed